### PR TITLE
Fix/28

### PR DESCRIPTION
**Pull request by GianlucaGuarini**
This pull request should fix the ie issues but It must be reviewed because it can contain breaking changes. I don't know why there is one test always failing and after debugging it for hours a couldn't find the issue. Please @cognitom can you have a look checking what's wrong? BTW The current unit test works only on Chrome/Phantomjs and we need to find a way to let it run also on firefox and the other browsers as well? I remember it was possible before

### CODE REVIEW

**Comment by cognitom** [comment]
@GianlucaGuarini thanks! I'm looking this PR.

File: test/specs/core.specs.js [hunk 1]
```diff
@@ -47,8 +47,6 @@ describe('Core specs', function() {
         e.preventDefault()
       })
     })
-    // start router
-    route.start()
```
Comment (cognitom) [inline test/specs/core.specs.js]: @GianlucaGuarini why you remove this line?
This causes "detects link clicked" failure.

Comment (cognitom) [inline test/specs/core.specs.js]: Hmm... still failing after adding this line again. I'll look into it deeper.

Comment (cognitom) [inline test/specs/core.specs.js]: Ah, I gotcha. The problem is `setTimeout`. We need some waits in the test, maybe. I'll try it.

**Comment by cognitom** [comment]
@GianlucaGuarini now this PR passes all tests :-)
Then we got to set up browserstack?

**Comment by GianlucaGuarini** [comment]
awesome @cognitom! You can publish the new release on npm we will fix this issue later.. I would like to make a new riot release this weekend

**Comment by cognitom** [comment]
OK, done. Now v2.3.12 has just published on npm ;-)
I've added fix for #30, too.

**Comment by GianlucaGuarini** [comment]
@cognitom well done :)

**Pull request merged**